data_steward/cdr_cleaner/clean_cdr_engine.py

```diff
@@ -77,28 +92,40 @@ def clean_dataset(project=None, statements=None, data_stage=stage.UNSPECIFIED):
                                      batch=batch)
```

Comment:
OPTIONAL:  At line 85, it logs which query is executing.  It would be nice to change this to something like:
```
statement_length = len(statements)
for index, statement in enumerate(statements):
    ...
    LOGGER.info("Running query %d out of %d:\n%s", index, statement_length, rule_query)
```

Reply:
This is done.